meson.build

```
@@ -12,6 +12,12 @@ python3_dep = dependency('python3', required : get_option('enable_python'))
 py3_inst = import('python').find_installation('python3')
 py3 = py3_inst
 pybind11_dep = dependency('pybind11', required : get_option('enable_python'))
+# For pybind11, if version < 2.4.4 then we need to add -fsized-deallocation flag
+if pybind11_dep.found() and meson.get_compiler('cpp').get_id() == 'clang'
```

```suggestion
if pybind11_dep.found() and cc.get_id() == 'clang'
```

... but I don't know why it's called `cc` on line 7 for a cxx compiler...